scripts/dev/e2e.py

```diff
@@ -257,31 +257,31 @@ def parse_args():
 
 def build_and_push_images(args, dev_config):
     test_runner_name = dev_config.testrunner_image
-    if not args.skip_operator_install:
+    if not args.install_operator:
```

Comment:
yeah it's much more readable keeping it in the positive form

Reply:
It was actually misleading as we were installing the operator if it was specified to NOT install it. I changed everything to be totally in the positive form now